src/index.html

```diff
@@ -85,7 +86,12 @@ <h1 class="has-text-weight-bold">SankeyDiagram.net</h1>
 target two [5] target three
 target two [25] target four
 target two [20] target five
-target five [15] target six [lightgreen]</textarea>
+target five [15] target six [lightgreen]
+</textarea>
+                            <label for="import-text-input">
+                                <span class="button is-dark p-1 mb-1 has-shadow import-txt-button is-clickable level-item" data-tooltip="Copied!"><span class="material-icons mr-1">file_upload</span> Import from TXT</span>
```

Comment:
Also "data-tooltip="Copied!"" could be removed here, since it doesn't shop up anyways from what I've observed